dcap/types/src/tcb.rs

```diff
@@ -3,11 +3,8 @@
 //! TCB measurements for an SGX enclave.
 //!
 //! The TCB measurements are present as OID extensions on the leaf PCK
-//! certificate.
-//! The OID extensions are listed at
-//! <https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/4cb5c8b81f126f9aa3ee921d7980a909a9bd676d/QuoteVerification/QVL/Src/AttestationParsers/src/ParserUtils.h#L57>.
-//! There doesn't appear to be a more formal place where these OID extensions
-//! are documented .
+//! certificate. The extensions are documented in
+//! <https://api.trustedservices.intel.com/documents/Intel_SGX_PCK_Certificate_CRL_Spec-1.5.pdf>
```

Comment:
Not a strong need for this change, but I figured I should link to the official document now that I realized which one has it.